requirements.txt

```diff
@@ -1,10 +1 @@
-blinker==1.4
-flask-login==0.4.1
-flask-oauthlib==0.9.5
-flask-restful==0.3.6
-flask-wtf==0.14.2
-flask==1.0.2
-jinja2==2.10
-pyjwt==1.6.4
-unidecode==1.0.22
-webargs==4.1.2
+
```

Comment:
Why not deleting the whole file?

Reply:
It was a shortcut. I expect the right thing to do is to remove the python reqs installation from gulp, but I've never edited gulp config before.

Reply:
In a subsequent commit I have now taken out pip install requirements from gulp/run.py and removed requirements.txt.